<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransmissionMode</class>
 <widget class="QWidget" name="TransmissionMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transmission mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="offButton">
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="continousTransmissionButton">
        <property name="text">
         <string>Continous transmission</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pushToTalkButton">
        <property name="text">
         <string>Push-to-Talk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="toggleModeButton">
        <property name="text">
         <string>Toggle mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
